ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -106,13 +105,22 @@ const syncAndReloadTraining = fp.flow(
 
 // Helper Functions
 
+export const isExpiring = (expiration?: number): boolean =>
```

Comment:
new version accounts for undefined: unlike before, this does not count as expiring.